dotCMS/src/main/java/com/dotmarketing/filters/CMSUrlUtil.java

```diff
@@ -530,4 +532,15 @@ public String getQueryStringFromUri(final String uri) {
 				uri.substring(indexOf+1):
 				null;
 	}
+
+	public static String getCurrentURI(final HttpServletRequest request)  {
+		try {
+			return URLDecoder.decode((request.getAttribute(Constants.CMS_FILTER_URI_OVERRIDE) != null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Constants.CMS_FILTER_URI_OVERRIDE' due to existing static import 'com.dotmarketing.filters.Constants.CMS_FILTER_URI_OVERRIDE'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4842460)